src/Math-Numerical/PMFisherTippettDistribution.class.st

```diff
@@ -68,12 +68,12 @@ PMFisherTippettDistribution >> distributionValue: aNumber [
 
 { #category : #initialization }
 PMFisherTippettDistribution >> initialize: aNumber1 scale: aNumber2 [
-		"Private - Initialize the parameters of the receiver."
+	"Initialize the parameters of the receiver."
+
 	aNumber2 > 0
-		ifFalse: [ self error: 'Illegal distribution parameters'].
+		ifFalse: [ self error: 'Illegal distribution parameters' ].
```

Comment:
Here, for me, it would be clearer if we could invert the clause and say

`number < 0 ifTrue: [...]`

Reply:
It would be ``aNumber2 <=0 ifTrue: [...]`` I guess.
Why clearer?